tests/e2e/c/consume_gas.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity = 0.8.6;
+
+contract ConsumeGas {
+  function run() public {
+    while (true){}
```

Comment:
I'm partial to hand-writing EVM bytecode to accomplish this goal, but I guess this is fine 😉